src/inline-svg.directive.ts

```diff
@@ -141,7 +141,7 @@ export class InlineSVGDirective implements OnInit, OnChanges, OnDestroy {
   private _insertEl(el: Element) {
     if (this.replaceContents && !this.prepend) {
       if (this._prevSVG) {
-        this._prevSVG.remove();
+        this._el.nativeElement.removeChild(this._prevSVG);
```

Comment:
Do you think it'd be better to check if `remove` exists and fallback to this? It'd be nice to use it if it's not IE.